code/modules/clothing/head/jobs.dm

```diff
@@ -605,6 +605,7 @@
 	desc = "Used by doctors to look into a patient's eyes, ears, and mouth. \
 		A little useless now, given the technology available, but it certainly completes the look."
 	icon_state = "headmirror"
+	clothing_traits = list(TRAIT_SURGICAL_TRAINING)
```

Comment:
why does the head mirror get the trait, but not any of the surgical caps?

Reply:
realism: head mirror lets you see what you're doing better
supply: there are slightly fewer head mirrors available without restocking
funny: head mirror are hugely outdated in the far flung future of ss13 so wearing one makes you look like a lunatic quack